examples/vue/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "validate-field-with-vue",
+  "version": "1.0.0",
+  "description": "Example integrating fonk with Vue.js 2 at field level.",
+  "keywords": [
+    "fonk",
+    "form-validation",
+    "form",
+    "vue"
+  ],
+  "scripts": {
+    "start": "vue-cli-service serve",
+    "build": "vue-cli-service build"
+  },
+  "dependencies": {
+    "@lemoncode/fonk": "^1.1.0",
```

Comment:
Remove `^` to fix dependencies please, we have some issues with cached codesanboxes